### PR TITLE
SGD8-1506: Fixed hidden & pubdate attributes.

### PR DESCRIPTION
**Pull request by pj-claeys**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Review by delrueba — COMMENTED** [review]
Accordion JS will revert hidden back to true.

**Comment by MPParsley** [comment]
> Accordion JS will revert hidden back to true.

Then we should fix Accordion JS accordingly, no?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All Notable changes to `digipolisgent/drupal_theme_gent-base`.
 
+## [Unreleased]
+
+### Removed
+
+* Removed pubdate attribute in time tag - supported any more.
```
Comment (MPParsley) [inline CHANGELOG.md]:
```suggestion
* Removed unsupported pubdate attribute in time tag.
```

**Review by MPParsley — APPROVED** [review]

**Comment by MPParsley** [comment]
@CaliJens, kan je dit conflict resolven en dan mag je dit mergen naar develop.

**Review by MPParsley — APPROVED** [review]

**Pull request merged**